.gitignore

```diff
@@ -1 +1,3 @@
-.DS_Store
\ No newline at end of file
+.DS_Store
+.DS_Store
+.DS_Store
```

Comment:
I'm not sure if you added these, but you only need to gitignore a file once.

Reply:
I have added that file before and yet again it keeps coming up whenever i make changes to my other files.

Reply:
It's probably the program you use to code that is automatically generating it. Don't worry about it.